libmscore/excerpt.cpp

```diff
@@ -886,6 +886,10 @@ void Excerpt::cloneStaff(Staff* srcStaff, Staff* dstStaff)
                               ncr->lyrics().clear();
 
                               for (Element* e : seg->annotations()) {
+                                    if (!e) {
+                                          qDebug("cloneStave: corrupted annotation found.");
```

Comment:
All occurrences of Stave were recently changed to Staff, and similarly from Staffs to Staves.
Plus, the name of the function is `cloneStaff`, not `cloneStave`, so this indication is misleading.

Reply:
You are right. Line is just copied which had the same mistake. I'll correct them all.

Reply:
Done